config/ingest.py

```diff
@@ -19,11 +21,13 @@
 
 config.register.unique = ['visit']
 config.register.columns = {
-    'visit': 'int',
-    'basename': 'text',
-    'filter': 'text',
-    'date': 'text',
-    'expTime': 'double',
-    'object': 'text',
+    'visit':      'int',
+    'basename':   'text',
+    'filter':     'text',
+    'date':       'text',
+    'expTime':    'double',
+    'object':     'text',
+    'imgType':    'text',
+    'wavelength': 'double',
 }
```

Comment:
We do not align because the diffs look terribly confusing when you add a new key that is longer anywhere to the dict. PEP8 explicitly says not to align variable definitions because of this, and I believe this code will trigger an E231 error. Has this code been checked with `flake8`?